sql/core/src/test/scala/org/apache/spark/sql/CachedTableSuite.scala

```diff
@@ -512,6 +512,9 @@ class CachedTableSuite extends QueryTest with SQLTestUtils
    * Verifies that the plan for `df` contains `expected` number of Exchange operators.
    */
   private def verifyNumExchanges(df: DataFrame, expected: Int): Unit = {
+    withSQLConf(SQLConf.AUTO_BROADCASTJOIN_THRESHOLD.key -> "-1") {
+      df.collect()
+    }
```

Comment:
avoid AQE convert first acess join to bhj

Reply:
Can we add a comment to explain why we need to force shuffle join here?

Reply:
oh, I missed this comment. You will find this method only checks the `ShuffleExchangeExec` if you are looking at the next line. So we should force shuffle join here to keep the original test idea.